frontend/src/scenes/session-recordings/playlist/sessionRecordingsPlaylistLogic.ts

```diff
@@ -417,7 +429,14 @@ export const sessionRecordingsPlaylistLogic = kea<sessionRecordingsPlaylistLogic
             'start_time' as RecordingsQuery['order'],
             { persist: true },
             {
-                setOrderBy: (_, { orderBy }) => orderBy,
+                setOrderBy: (_, { orderBy }) => orderBy as RecordingsQuery['order'],
+            },
+        ],
+        randomSample: [
```

Comment:
I don't think I get the use for random sample (especially since it appears to be a persistent randomness when toggling the sampling on and off)

Reply:
Original thinking was in https://github.com/PostHog/posthog/issues/8182

Tbh I might just remove it for now because I can foresee it causing customer confusion if there's misunderstanding about it and it hasn't made it past the PR 😅

Reply:
well, top and right not front and center 🤣